src/Tests/PostfinancePaymentTest.php

```diff
@@ -115,30 +131,63 @@ function testPostfinanceAcceptPayment() {
     \Drupal::state()->set('postfinance.testing', TRUE);
 
     // Load payment configuration.
-    $payment_config = \Drupal::config('payment_postfinance.settings');
+    $payment_config = $this->config('payment_postfinance.settings');
 
     // Check if payment link is correctly set.
     $this->assertEqual($payment_config->get('payment_link'), $GLOBALS['base_url'] . Url::fromRoute('postfinance_test.postfinance_test_form')->toString());
```

Comment:
this assertion doesn't make sense, we set this ourself, we don't have to test our own test.